circle.yml

```diff
@@ -4,4 +4,4 @@ dependencies:
 test:
   override:
     - npm test
-    - ./node_modules/.bin/grunt shell:coveralls
+    - ./node_modules/.bin/grunt shell:coveralls --force
```

Comment:
The error message I was getting was here: https://circleci.com/gh/NREL/api-umbrella-gatekeeper/70

The `--force` flag "fixed", at least in the sense of allowing the tests to pass. I'll try removing it and activating coveralls on my repo.